client.go

```diff
@@ -2,17 +2,16 @@ package libovsdb
 
 import (
 	"crypto/tls"
-	"crypto/x509"
 	"encoding/json"
```

Comment:
could we have standare for import package here:
3 section:
1st section: default go package
2nd section: ebay dependancy.
3rd section: go-ovn dependancy.

Reply:
i can't because this is goimports and gofmt it always sort standard and after it all other with alphabetical order